extension/src/popup.jsx

```diff
@@ -64,13 +65,18 @@ const updateRating = (id, rating) => ({
   id: id,
   value: rating
 });
+const sayThanks = ratings_length => ({
+  type: SAY_THANKS,
+  ratings_length
```

Comment:
It is slightly confusing to use `ratings_length` here when we've used `ratings_count` everywhere else.

Reply:
fixed in 591236d